src/design-tokens/design-tokens.stories.mdx

```diff
@@ -0,0 +1,221 @@
+import { Meta, ColorPalette, ColorItem } from '@storybook/addon-docs/blocks';
+import * as brandTokens from './brand.yml';
+
+<Meta title="Design Tokens"/>
+
+# Design Tokens
+
+## Primary Colors
+These are our core colors. Blue is our brand color and is used by many elements like buttons, links, backgrounds, and illustration.
+<ColorPalette>
+  <ColorItem
+    title="$blue-3"
+    colors={[brandTokens.blue3]}
+  />
+  <ColorItem
+    title="$blue-5"
+    colors={[brandTokens.blue5]}
+  />
+  <ColorItem
+    title="$gray-1"
+    colors={[brandTokens.gray1]}
+  />
+</ColorPalette>
+
+## Secondary Colors
+
+Our secondary palette consists of green, purple, fuchsia, and orange. These colors are meant to highlight and compliment our primary brand colors.
+
+<ColorPalette>
+  <ColorItem
+    title="$green-3"
+    colors={[brandTokens.green3]}
+  />
+  <ColorItem
+    title="$purple-3"
+    colors={[brandTokens.purple3]}
+  />
+  <ColorItem
+    title="$fuchsia-3"
+    colors={[brandTokens.fuchsia3]}
+  />
+  <ColorItem
+    title="$orange-3"
+    colors={[brandTokens.orange3]}
+  />
+</ColorPalette>
+
+## Extended Color Palette
+The extended palette is intended to be ever evolving, it offers an additional range of colors work with. Each color has 5 values, with 1 being the lightest and 5 being the darkest. The number of different values was determined by cataloging the current use of color throughout our site to create a sufficient number of variations. Usage of these colors will vary, but they should come in handy for illustrations and components.
+
+<ColorPalette>
+  <ColorItem
+    title="$blue-1"
+    subtitle="Accessible on the darkest colors in our palette."
+    colors={[brandTokens.blue1]}
```

Comment:
I'm curious if you tried organizing these in an array from darkest to lightest? `{[brandTokens.blue1, brandtokens.blue2, ....]}`

I found this arrangement helpful from a design standpoint. We could highlight accessible color combinations in its own section?

Reply:
I did try that approach and I like that it condenses the information, however I didn't see a way to indicate the token for each color. Only the rgb values are visible beneath each swatch.

Reply:
You could potentially set `title` to `blue-*` and subtitle to `Steps 1 through 5` (or whatever convention we go with). But this isn't a blocker.